utils/clone-mlir.sh

```diff
@@ -1,3 +1,3 @@
 git clone -n https://github.com/llvm/llvm-project.git
 # Check out a specific branch that is known to work with ONNX-MLIR.
-cd llvm-project && git checkout 01d233ff403823389f8480897e41aea84ecbb3d3 && cd ..
+cd llvm-project && git checkout af20aff35ec37ead88903bc3e44f6a81c5c9ca4e && cd ..
```

Comment:
@AlexandreEichenberger Check here please

Reply:
Sorry, I see it now :-)